python/ray/_common/tests/test_synchronization_actors.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
```

Comment:
IMO let's keep the name as `test_utils` to signal that this code is only meant to be used in testing